src/resolve/resolved-gperf.gperf

```diff
@@ -32,3 +32,4 @@ Resolve.ReadEtcHosts,              config_parse_bool,                    0,
 Resolve.ResolveUnicastSingleLabel, config_parse_bool,                    0,                   offsetof(Manager, resolve_unicast_single_label)
 Resolve.DNSStubListenerExtra,      config_parse_dns_stub_listener_extra, 0,                   offsetof(Manager, dns_extra_stub_listeners)
 Resolve.CacheFromLocalhost,        config_parse_bool,                    0,                   offsetof(Manager, cache_from_localhost)
+Resolve.ServeStale,                config_parse_bool,                    0,                   offsetof(Manager, enable_serve_stale)
```

Comment:
I think this should take a time vlaue, and specify the hard TTL when we'll stop serving stale data. And if people set this to "0" then we will disable the feature, and if they set it to "infinity" then we'll keep the stale data forever.

config_parse_usec already parses "infinity" for you, hence this shouldn't be too hard.